composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -37,8 +37,9 @@ def initialize(extensions)
             (?<=require\s)php(?:\-[^\s\/]+)?\s.*?\s(?=->) # composer v2
           }x
         VERSION_REGEX = /[0-9]+(?:\.[A-Za-z0-9\-_]+)*/
-        SOURCE_TIMED_OUT_REGEX =
-          /The "(?<url>[^"]+packages\.json)".*timed out/
```

Comment:
We no longer have to match this error format, as it's leftover from `composer v1`... 

Instead I manually triggered the test and checked the actual error message currently returned by `composer v2`... see the code comment.